entries/p/portu.cz.json

```diff
@@ -0,0 +1,16 @@
+{
+    "Portu": {
+        "domain": "portu.cz",
+        "url": "https://www.portu.cz",
```

Comment:
```suggestion
```

Because you can go directly to `https://portu.cz`, you don't need the `url`.

Reply:
It 301 redirects to `www.` but fair enough.